website/content/docs/platform/k8s/helm/examples/ha-tls.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: 'docs'
+page_title: 'HA Cluster with TLS'
+sidebar_current: 'docs-platform-k8s-examples-ha-tls'
+description: |-
+  Describes how to set up a Raft HA Vault cluster with TLS certificate
+---
+
+# Raft HA Server with TLS 
+
+Follow the steps from the example [HA Vault Cluster with Integrated Storage](/docs/platform/k8s/helm/examples/ha-with-raft) to build the cluster.
+
+Follow the examples and instructions in [Standalone Server with TLS](/docs/platform/k8s/helm/examples/standalone-tls) to create a TLS certificate.
+
+Before cluster initialization and without proper configuration, the following warning is to be expected:
+```shell
+core: join attempt failed: error="error during raft bootstrap init call: Put "https://vault-${N}.${SERVICE}:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate is valid for ${SERVICE}, ${SERVICE}.${NAMESPACE}, ${SERVICE}.${NAMESPACE}.svc, ${SERVICE}.${NAMESPACE}.svc.cluster.local, not vault-${N}.${SERVICE}"
+```
+
+The concepts for [Integrated Storage and TLS](/docs/concepts/integrated-storage#integrated-storage-and-tls) elaborate three possible solutions to mitigate this TLS verification warning and bootstrap the Raft cluster, two of which are discussed in further detail here.
+
+The warning disappears when the the [expected TLS servername](/docs/concepts/integrated-storage#autojoin-with-tls-servername) is correctly configured using [`leader_tls_servername`](/docs/configuration/storage/raft#leader_tls_servername) in the Raft stanza (`${CN}` in the example below):
+
+```hcl
```

Comment:
It might be useful to highlight the important line for folks so they know which part of the Raft config you're talking about. You can highlight the relevant line by wrapping the `CodeBlockConfig` component around your code example (with whitespace before/after each tag) and noting the specific line(s) you want to highlight. For example:

```
<CodeBlockConfig highlight="6,14,22">

YOUR_CODE_BLOCK

</CodeBlockConfig>
```